packages/tools/kadena-cli/src/utils/logger.ts

```diff
@@ -77,7 +77,7 @@ export const defaultTransport: Transport = (record) => {
     [LEVELS.info]: stdErrChalk.white,
```

Comment:
still doubting these color levels, and maybe use / create something for the inversion? when using black / white terminal. 

Maybe just black on white bg, and white on black bg